text/0002-rfc-process.md

```diff
@@ -0,0 +1,469 @@
+feature as a pull request to the relevant repo. Being "active" is not a rubber
+stamp, and in particular still does not mean the feature will ultimately be
+merged; it does mean that in principle all the major stakeholders have agreed
+to the feature and are amenable to merging it.
+
+Furthermore, the fact that a given RFC has been accepted and is "active"
+implies nothing about what priority is assigned to its implementation, nor does
+it imply anything about whether a developer has been assigned the task of
+implementing the feature. While it is not *necessary* that the author of the
+RFC also write the implementation, it is by far the most effective way to see
+an RFC through to completion: authors should not expect that other project
+developers will take on responsibility for implementing their accepted feature.
+
+Modifications to "active" RFCs can be done in follow-up pull requests. We
+strive to write each RFC in a manner that it will reflect the final design of
+the feature; but the nature of the process means that we cannot expect every
+merged RFC to actually reflect what the end result will be at the time of the
+next release.
+
+In general, once accepted, RFCs should not be substantially changed. Only very
+minor changes should be submitted as amendments. More substantial changes
+should be new RFCs, with a note added to the original RFC. Exactly what counts
+as a "very minor change" is up to the team to decide.
+
+## Reviewing RFCs
+[Reviewing RFCs]: #reviewing-rfcs
+
+While the RFC pull request is up, the core developers may schedule meetings with
+the author and/or relevant stakeholders to discuss the issues in greater detail,
+and in some cases the topic may be discussed at a regular LF meeting. In either
+case a summary from the meeting will be posted back to the RFC pull request.
+
+Final decisions about RFCs are made after the benefits and drawbacks are well
+understood. These decisions can be made at any time. When a decision is made,
+the RFC pull request will either be merged or closed. In either case, if the
+reasoning is not clear from the discussion in the thread, the a comment
+describing the rationale for the decision will be added.
+
+
+## Implementing an RFC
+[Implementing an RFC]: #implementing-an-rfc
+
+Some accepted RFCs represent vital features that need to be implemented right
+away. Other accepted RFCs can represent features that can wait until some
+arbitrary developer feels like doing the work. Every accepted RFC has an
+associated issue tracking its implementation in the relevant repository.
+
+The author of an RFC is not obligated to implement it. Of course, the RFC
+author (like any other developer) is welcome to post an implementation for
+review after the RFC has been accepted.
+
+If you are interested in working on the implementation for an "active" RFC, but
+cannot determine if someone else is already working on it, feel free to ask
+(e.g. by leaving a comment on the associated issue).
```

Comment:
Is this really necessary to document this for our RFC workflow?

I think the interesting part for the implementation would be how issues, PRs, or commits might be associated to an RFC, should the RFC number be used for this? If yes how?

Reply:
Each active RFC should have a tracking issue (that itself may contain smaller issues/tasks). Commits and PRs that implement the feature completely or provide increments towards a complete implementation should reference the tracking issue. (This isn't any different from how we handle it how, so probably we don't need to mention it explicitly)